### ISSUE TITLE
The training EPE error explodes on FlyingChairs dataset.

### ISSUE THREAD

**Issue by PkuRainBow** [NVIDIA/flownet2-pytorch#7]
Here I train the flownet2 directly on the flying chairs. However, the EPE error explode after 7 epochs.

We can find that the EPE error is always increasing during the training procedure.

![image](https://user-images.githubusercontent.com/4639578/34023133-2b52ddb0-e17e-11e7-8c2b-06c1a9838422.png)

**Comment by fitsumreda** [comment]
The Flownet2 paper actually recommends to train the networks one at a time, starting with FlowNetC, then  fix this, and train FlowNetS1 and so on.
training flownet2 directly on flying-chairs would lead to the type of epe explosion you're getting.

**Comment by PkuRainBow** [comment]
@fitsumreda But I just try to train the FlowNetC on FlyingChairs directly, I meet other problems. Could you help me?

![image](https://user-images.githubusercontent.com/4639578/34029629-78a9fc08-e1a4-11e7-90fe-6561dfabd117.png)

**Comment by fitsumreda** [comment]
Could you run this command that runs flownetc, and see if you get the same error?
python main.py --batch_size 8 --model FlowNet2C --optimizer=Adam --optimizer_lr=1e-4 --loss=MultiScale --loss_norm=L1 \
--loss_numScales=5 --loss_startScale=4 --optimizer_lr=1e-4 --crop_size 384 512 \
--training_dataset FlyingChairs --training_dataset_root /path/to/flying-chairs/dataset  \
--validation_dataset MpiSintelClean --validation_dataset_root /path/to/mpi-sintel/clean/dataset

**Comment by PkuRainBow** [comment]
It works! I am wondering whether you could provide more details, such as the well trained models by yourself.

**Comment by dvorak0** [comment]
@PkuRainBow Did get converged result with the mentioned training parameters?

Using `python main.py --batch_size 8 --model FlowNet2C --optimizer=Adam --optimizer_lr=1e-4 --loss=MultiScale --loss_norm=L1 \
--validation_frequency 1 \
--loss_numScales=5 --loss_startScale=4 --optimizer_lr=1e-4 --crop_size 384 512 \
--training_dataset FlyingChairs --training_dataset_root /root/per_nas/FlyingChairs_release/data \
--validation_dataset MpiSintelClean --validation_dataset_root /root/MPI-Sintel-complete/training
` after 1 epoch, my training loss curve keeps jumping.

**Comment by MatthewD1993** [comment]
@dvorak0 Do you get good result on training FlowNet2C? My curve is also bumping seriously while training. Very grateful if you could share your experience. Thanks!

**Comment by qiuzhongwei-USTB** [comment]
@PkuRainBow hi, have you trained flownet2 successfully by training part of flownet2 first? could you tell me how to freeze parameters?

**Comment by Queenyy** [comment]
@MatthewD1993 Hi, have you trained FlowNet2C successfully? My loss curve is also bumping while training. Thanks a lot.
